backend/src/packages/journal-entries/journal-entry.service.ts

```diff
@@ -31,19 +50,57 @@ class JournalEntryService implements Service {
 
   public async create(
     payload: JournalEntryCreateRequestDto,
+    userId: number,
   ): Promise<JournalEntryGetAllItemResponseDto> {
+    const user = await userService.findById(userId);
+
+    if (!user) {
+      throw new UsersError({
+        status: HTTPCode.NOT_FOUND,
+        message: ExceptionMessage.USER_NOT_FOUND,
+      });
+    }
+
     const item = await this.journalEntryRepository.create(
-      JournalEntryEntity.initializeNew({
-        title: payload.title,
-        text: payload.text,
+      JournalEntryEntity.initialize({
+        id: null,
+        userId,
+        createdAt: null,
+        updatedAt: null,
+        title: sanitizeInput(payload.title),
+        text: sanitizeInput(payload.text),
```

Comment:
why we don't use `initializeNew` here? You already added the `useId` as a property there

Reply:
You are right, thanks for noticing.